src/test/java/longtimenosee/model/person/predicate/RiskAppetiteMatchesInputPredicateTest.java

```diff
@@ -0,0 +1,59 @@
+package longtimenosee.model.person.predicate;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import longtimenosee.testutil.PersonBuilder;
+
+public class RiskAppetiteMatchesInputPredicateTest {
+    @Test
+    public void test_equals() {
+        String firstPredicateInput = "H";
+        String secondPredicateInput = "M";
+
+        RiskAppetiteMatchesInputPredicate firstPredicate = new RiskAppetiteMatchesInputPredicate(firstPredicateInput);
+        RiskAppetiteMatchesInputPredicate secondPredicate = new RiskAppetiteMatchesInputPredicate(secondPredicateInput);
+
+        // EP: null value
+        assertFalse(firstPredicate.equals(null)); // Boundary value
+
+        // EP: same object
```

Comment:
In general, very good use of Equivalence Partitions! Good to see the team putting into practice what we learned during tutorials.